### ISSUE TITLE
Tips & Tricks section?

### ISSUE THREAD

**Issue by atifaziz** [amis92/csharp-source-generators#4]
What are your thoughts on adding a Tips & Tricks section? Reason I ask is that I knew it was possible to see the files generated by a source generator, but it took me 20 minutes to find the information on the web when I needed it (and I'd like to think my Google search Fu isn't too bad)…

```xml
<EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
<CompilerGeneratedFilesOutputPath>$(BaseIntermediateOutputPath)Generated</CompilerGeneratedFilesOutputPath>
```

This will eventually make its way into [the docs](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-9#support-for-code-generators) proper, but meanwhile, we need to help each other out. 😉 

To keep it simple and brief, such a section should only list tips and tricks that can fit in a tweet, [like one I just did (really as a note-to-self) about the above](https://twitter.com/raboof/status/1328426892882550784). What do you think?

**Comment by ironcev** [comment]
I like the proposal and would love to have such a section. Generators are still rather new and it will take time until enough material is out there for search engines to pick them up.

**Comment by amis92** [comment]
I like the idea. Go ahead and add the above into your PR! I think it fits somewhere after main Generator list, what do you think?

I should also add ToC I think...

**Comment by atifaziz** [comment]
Closed via a8471e6f80a65493b970be3f1bd4b29d5b7a6f26 (merge of RP #5).
